hack/update-codegen-dockerized.sh

```diff
@@ -44,9 +44,21 @@ ANTREA_CODEGEN_PATH=/go/src/antrea.io/antrea
 git clone ${ANTREA_SRC_PATH} ${ANTREA_CODEGEN_PATH}
 pushd ${ANTREA_CODEGEN_PATH}
 
-# Remove all auto-generated files
-git grep --files-with-matches -e 'Code generated by .* DO NOT EDIT' pkg | xargs rm
-git grep --files-with-matches -e 'This file was autogenerated by go-to-protobuf' pkg | xargs rm
+function cleanup {
+  # Copy generated code back into the source repository, which was cloned at the
+  # beginning of this script to make a temporary working copy.
+  git ls-files --modified --others --exclude-standard | while read file; do
+      if [ -e $file ]; then
+          cp $file ${ANTREA_SRC_PATH}/$file
+      else
+          rm ${ANTREA_SRC_PATH}/$file
+      fi
+  done
+  popd
+  rm -rf ${ANTREA_CODEGEN_PATH}
```

Comment:
I feel this should be out of cleanup function since `pushd` is out of this.

Reply:
Thanks. I switched to using a trap for cleanup, and the function has been renamed to `copy_generated_code_to_source` and only takes care of copying generated files as needed.